datasets/_global/gem_energy_ownership/crawler.py

```diff
@@ -74,10 +77,32 @@ def crawl_company(context: Context, row: Dict[str, str], skipped: Set[str]):
     entity = context.make(schema)
     entity.id = context.make_slug(id)
 
+    names_to_lookup = [full_name, original_name]
+    associates = set()
+    for name in names_to_lookup:
+        result = context.lookup("associates", name)
+        if result and result.associates:
+            associates.update(result.associates)
```

Comment:
is there anything in the data we can use to determine when we should expect an associates lookup to exist? e.g. `"(" in name and "," in name` or something, and then warn? Not sure how many false positives we'll get, and I don't think it's worthwhile for one or two in 1000. But try it - maybe we spot more instances where we need to map out related entities as opposed to aliases.